governance/executive-council.md

```diff
@@ -0,0 +1,262 @@
+This process ensures the Council has the skills and expertise necessary to meet
+the evolving pyOpenSci needs.
+
+### Executive Council selection process
+
+
+1. **Identify organization needs**:
+   - The Council first evaluates its current composition to identify areas where
+     additional expertise, perspectives, or skills could be useful to support the
+     organization’s mission and strategy.
+
+2. **Identify Candidates**:
+   - Potential candidates are identified based on their experience, expertise,
+     and alignment with pyOpenSci’s mission, vision, and values. This may include
+     leaders in the open science community, individuals with financial or
+     governance expertise, or others with relevant nonprofit governance skills.
+
+3. **Review and nominate the new member**:
+   - Candidates are discussed and nominated by current Council members during a
+     meeting or through the Executive Council Slack channel. After nomination, the Executive Director
+     will contact the candidate to see if they are interested/available.
+
+5. **Onboard the new candidate**:
+   - Once approved, new members are onboarded following the onboarding process
+     outlined in this document.
+
+
+### Terms and evaluation
+
+- Executive Council members are expected to serve for 2–3 years.
+- At the end of each 2-year term, members may choose to step down or renew their
+  commitment based on mutual agreement with the Council.
+
+As needed, the Executive Council may conduct periodic evaluations to ensure its composition continues
+to meet the organization’s needs.
+
+## Onboarding and offboarding Executive Council members
+
+The onboarding and offboarding of Executive Council (EC) members ensure:
+
+* smooth transitions,
+* that new members are set up for success and
+* that departing members are appropriately transitioned out of their roles.
+
+### Onboarding a new Executive Council member
+
+When a new member joins the Executive Council, the following steps should be
+completed to integrate them into the team:
+
+1. **Access to tools and resources**:
+   - Add the new member to the pyOpenSci Executive Council shared [Google Drive](google-drive).
+     This drive is where we share meeting notes,
+     agendas, budgets and other resources.
+   - Add the new member to the Executive Council Slack channel, the primary
+     communication platform for the Council.
+
+2. **Orientation**:
+   - Provide a link to this Handbook, which outlines the Council’s
+     responsibilities, decision-making processes, and operational policies.
+   - Schedule an initial orientation meeting, if needed, to familiarize the new member
+     with pyOpenSci’s mission, vision, values, and ongoing initiatives.
+
+3. **Notifications**:
+   - Notify the pyOpenSci's Fiscal Sponsor (Community Initiatives) of any changes in roles
+     that impact the chair position or budget oversight responsibilities.
+
+4. **Welcome and integration**:
```

Comment:
to the council, and to the pyOS community?

Reply:
Good point -- i've clarified this. 


4. **Welcome and integration**:
   - Introduce the new member to the Executive Council during a meeting and to the pyOpenSci community through Slack
     announcement. Highlight their expertise and how it complements the team’s
     needs.